app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchEvent.kt

```diff
@@ -9,8 +9,7 @@ import org.simple.clinic.home.overdue.search.OverdueSearchQueryValidator.Result
 import org.simple.clinic.platform.util.RuntimePermissionResult
 import org.simple.clinic.util.RequiresNetwork
 import org.simple.clinic.widgets.UiEvent
-import java.util.Optional
-import java.util.UUID
+import java.util.*
```

Comment:
Can we revert this import change, please? It doesn't fit with our code formatting rules.